Source/JavaScriptCore/dfg/DFGSpeculativeJIT.cpp

```diff
@@ -16748,6 +16748,52 @@ void SpeculativeJIT::compileNumberIsNaN(Node* node)
     }
 }
 
+void SpeculativeJIT::compileNumberIsFinite(Node* node)
+{
+    switch (node->child1().useKind()) {
+    case DoubleRepUse: {
+        SpeculateDoubleOperand argument(this, node->child1());
+        GPRTemporary scratch(this);
+        GPRReg scratchGPR = scratch.gpr();
+        FPRReg argumentFPR = argument.fpr();
+        FPRTemporary diff(this);
+        FPRReg diffFPR = diff.fpr();
+        FPRTemporary neg(this);
+        FPRReg negFPR = neg.fpr();
```

Comment:
Let's follow to the typical procedure, first defining all operands and temporaries, and then getting gpr() etc.
```
SpeculateDoubleOperand argument(this, node->child1());
GPRTemporary scratch(this);
FPRTemporary diff(this);
FPRTemporary neg(this);

FPRReg argumentFPR = argument.fpr();
GPRReg scratchGPR = scratch.gpr();
FPRReg diffFPR = diff.fpr();
FPRReg negFPR = neg.fpr();
```

Reply:
Thanks. Fixed. Is it preferable to use`add + neg` over `sub`?